### DISCUSSION TITLE
Does this work for Ventura 13.5.2?

### DISCUSSION THREAD

**Discussion by timfea926** [halo/LinkLiar#61] [Q&A] [answered]
I noticed the last time this app was updated was in 2021 for Big Sur. I hesitate to download it as I'm uncertain whether it will work for Ventura...any insights to this?

**Comment by halo** [comment, marked as answer]
For LinkLiar it doesn't matter much whether you're using Big Sur or Ventura. Keep in mind, that LinkLiar only calls `sudo ifconfig en0 ether aa:bb:cc:dd:ee:ff`. As long as that command works (you can try it in your Terminal), then LinkLiar will work. It's only a tiny GUI that helps you run that command in a semi-automated manner.

Some MAC addresses traditionally never worked (ifconfig won't accept them). But since Ventura, the ifconfig command itself has become somewhat unreliable, which is a mystery to me.

A quick google would reveal plenty of dicussions such as these:
https://gist.github.com/thesauri/022a307234eb3296fae6487cacc1fc1f
https://www.reddit.com/r/MacOS/comments/yeb94t/ventura_mac_spoofing_no_longer_working

I basically throw my hands up into the air, as long as ifconfig doesn't reliably and predictably do what it should. There is unfortunately little I can do about that.

**Reply by timfea926** [reply]
Thank you Halo, that's all I needed. I am not a developer by any means (though I love researching and finding new technologies), so my understanding of any terminal-related coding is elementary at best. I appreciate you taking the time to answer my question.
